packages/components/pie-textarea/test/component/pie-textarea.spec.ts

```diff
@@ -93,92 +87,6 @@ test.describe('PieTextarea - Component tests', () => {
             });
         });
 
-        test.describe('size', () => {
```

Comment:
These tests were checking that the attributes were added to the element. As we're not styling with attributes anymore these aren't needed, and I didn't think the tests were useful enough to rewrite them to check for the class.